src/main/resources/db/migration/V2_8__add_aktivitetskrav_varsel_publisert.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE AKTIVITETSKRAV_VARSEL ADD COLUMN published_at TIMESTAMPZ;
```

Comment:
Heter det ikke `TIMESTAMPTZ`, altså TZ for timezone på slutten?

Reply:
Jo, riktig! Fikset.